src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx

```diff
@@ -169,19 +129,13 @@ const DocumentUploader = () => {
       <FileInput
         additionalClass={`${errorMsgClass} ${disabledOnEmptyDescClass}`}
         additionalErrorClass="vads-u-margin-bottom--1"
-        buttonText="Upload this document"
+        buttonText="Upload your document"
         onChange={onUploadFile}
         name="fileUpload"
         accept={FILE_TYPES.map(type => `.${type}`).join(',')}
-        errorMessage={errorMessage}
+        errorMessage={state.errorMessage}
       />
-      <button
-        className="vads-u-margin-top--3 usa-button"
-        onClick={onSubmit}
-        type="button"
-      >
-        Submit uploaded documents
-      </button>
+      <button onClick={onSubmit}>Submit files</button>
```

Comment:
We should add a `type` attribute to this button

Reply:
@jerekshoe Updated this